app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```

Comment:
Why this helper is exsist?

Reply:
Because I run rails generate controller so these files can generate. Next time I avoid to add in commit these files